.cascade-code/_cascade/formats/level/Masthead.vtl

```diff
@@ -0,0 +1,323 @@
+              </div>
+            #end
+          </div>
+          <div class="column header theme-bg-color">
+            <h2 class="header">$_EscapeTool.xml($brandedMastheadHeader)</h2>
+          </div>
+        </div>
+    </div>
+
+## Slider Masthead design added Nov 2016.
+#elseif ( $mastheadType == 'Slider - New' )
+    <!-- mastheadType: "Slider - New" -->
+    #set ( $slides = $_XPathTool.selectNodes($masthead, 'slider201611/slides/slide') )
+    #set ( $header = $_XPathTool.selectSingleNode($masthead, 'slider201611/header').value )
+    #set ( $firstSubtitle = $_XPathTool.selectSingleNode($slides[0], 'subTitle').value )
+
+  <div class="slider version-201611">
+    <div class="hidden slideOptions">
+        <div class="autoRotate">On</div>
+    </div>
+
+    <div class="grid">
+      <div class="column image">
+        <div class="rounded-slider">
+          <div class="rotatorContainer">
+            <div class="flex-container">
+              <div class="flexslider">
+                <ul class="slides">
+                  #foreach($slide in $slides)
+                    #set ( $slideSubtitle = $_XPathTool.selectSingleNode($slide, 'subTitle').value )
+                    #set ( $slideImagePath = $_XPathTool.selectSingleNode($slide, 'image/path').value )
+                    #set ( $slideImageAltText = $_XPathTool.selectSingleNode($slide, 'altText').value )
+                    #set ( $slidePhotoCaption = $_XPathTool.selectSingleNode($slide, 'photoCaption').value )
+
+                    <li class="slide">
+                      <img alt="${_EscapeTool.xml($slideImageAltText)}"
+                           src="[system-asset]${slideImagePath}[/system-asset]"/>
+
+                      ## Subtitle will be swapped out dynamically upon slide change.
+                      <input class="slideSubtitle"
+                             type="hidden"
+                             value="${_EscapeTool.xml($slideSubtitle).replace("\n", "<br/>")}"/>
+
+                      #if ($slidePhotoCaption != '')
+                        <div class="photo-by">
+                          ${_EscapeTool.xml($slidePhotoCaption)}
+                        </div>
+                      #end
+                    </li>
+                  #end
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="column header theme-bg-color">
+        <div class="aligned">
+          <h2 class="header">${_EscapeTool.xml($header).replace("\n", "<br/>")}</h2>
+          ## for ADA compliance, only show subheader if not empty
+          <div class="subheader">
+            ## HTML for subsequent slides comes from javascript
+            ## See app/assets/javascript/cascade/level/rotator.js
+            #if ( $firstSubtitle != '' )
+              <hr />
+              <h3>${_EscapeTool.xml($firstSubtitle).replace("\n", "<br/>")}</h3>
+            #end
+          </div>
```

Comment:
even this if statement can be moved to the javascript if we want. I just left it here because that's how it was before and added a comment about it.

Reply:
The `replace` code I recognize as my own. This was to accommodate a product owner request to be able to control line breaks in subtitles. 

I probably should have added a comment for it. If we do move it to js, let's definitely add comment.

Reply:
I'm indifferent when it comes to where we should put this conditional for the first slide, we can either leave it here or move it to the javascript. Idk if anyone has any strong opinions about it.